src/Symfony/Upgrade/Fixer.php

```diff
@@ -29,13 +30,30 @@ public function __construct(
         $this->stopwatch = $stopwatch;
     }
 
-    public function registerBuiltInFixers()
+    public function registerAllBuiltInFixers()
     {
+        /** @var AbstractFixer $fixer */
         foreach (new FixerIterator() as $fixer) {
             $this->addFixer($fixer);
         }
     }
 
+    /**
+     * @param array $fixerNameList
```

Comment:
Already in the code, brings just doubles maintenance.

Reply:
done